packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -115,7 +120,9 @@ class Chat extends Widget<Properties> {
   }
 
   _renderMessageList(): void {
-    const { items = [], user, showDayHeaders } = this.option();
+    const {
+      items = [], user, showDayHeaders, dayHeaderFormat, messageTimestampFormat,
```

Comment:
How do you think if we add line-break after each of options it will looks better?

Reply:
ed2574f9d5f3c5a301c284eb8acaba16c73321ad